TECH-OVERSIGHT-COMMITTEE.md

```diff
@@ -94,10 +94,11 @@ The current members of the TOC are shown below.
 
 | &nbsp;                                                         | Member           | Company    | Profile                                              | Term Start | Term End |
 | -------------------------------------------------------------- | ---------------- | ---------- | ---------------------------------------------------- | ---------- | --------
+| <img width="30px" src="https://github.com/davidhadas.png">     | David Hadas      | IBM        | [@davidhadas](https://github.com/davidhadas)         | 2023-09-12 | 2024     |
```

Comment:
@davidhadas pls check the details, email etc. Thanks!

Reply:
all is fine - thanks.